avalon/tools/widgets.py

```diff
@@ -120,6 +120,11 @@ def get_active_asset(self):
         current = self.view.currentIndex()
         return current.data(self.model.ObjectIdRole)
 
+    def get_active_asset_document(self):
+        """Return the asset id the current asset."""
+        current = self.view.currentIndex()
+        return current.data(self.model.DocumentRole)
```

Comment:
Solely added this to make it very trivial to retrieve the currently active asset's document as opposed to its name.